test/utils/v4HooksPoolsFiltering.ts

```diff
@@ -0,0 +1,50 @@
+import { V2SubgraphPool, V3SubgraphPool, V4SubgraphPool } from '@uniswap/smart-order-router'
+import { Hook } from '@uniswap/v4-sdk'
+import { HOOKS_ADDRESSES_ALLOWLIST } from '../../lib/util/hooksAddressesAllowlist'
+import { ChainId } from '@uniswap/sdk-core'
+
+export function routableHooks(pool: V4SubgraphPool): boolean {
```

Comment:
would rename this to 'isHookPoolRoutable(...)`

Reply:
yes